src/Storefront/Resources/views/storefront/component/address/address-form.html.twig

```diff
@@ -9,7 +9,8 @@
 
     {% block component_address_form_company %}
         {% if shopware.config.core.loginRegistration.showAccountTypeSelection %}
-            {% set accounTypeRequired = true %}
+            {% set accountTypeRequired = true %}
+            {# deprecated #}{% set accounTypeRequired = true %}
```

Comment:
have a look here https://github.com/shopware/platform/blob/master/src/Storefront/Resources/views/storefront/layout/header/header-minimal.html.twig#L4 for example. so this could be look like that
```suggestion
            {# @deprecated tag:6.4.0 - Variable "accounTypeRequired" will be removed without replacement. Use "accountTypeRequired" instead #}
            {% set accounTypeRequired = true %}
```